docs/reference/settings.rst

```diff
@@ -200,6 +200,15 @@ If a user has not uploaded a profile picture, Wagtail will look for an avatar li
 
 Changes whether the Submit for Moderation button is displayed in the action menu.
 
+Comments
+========
+
+.. code-block:: python
+
+    WAGTAILADMIN_COMMENTS_ENABLED = False
```

Comment:
Think we need something that indicates the default here. A comment like "Disable comments" just above this line will do

Reply:
Good idea, will add that!